src/redux/ui/navigation.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+import {
+  goBack,
+  push,
+} from 'react-router-redux'
+
+/*
+ * Thunk dispatched by any screen.
+ * Going back from anywhere
+ */
+export const navigateBack = () => (dispatch) => {
+  dispatch(goBack())
+}
+
+export const navigateToWallets = () => (dispatch) =>
+  dispatch(push('/wallets'))
```

Comment:
Maybe better to use path constants instead of navigateTo... actions. It can eliminate boilerplate code and increase flexibility